dubbo-common/src/main/java/org/apache/dubbo/rpc/model/ReflectionServiceDescriptor.java

```diff
@@ -89,6 +98,21 @@ private void initMethods() {
                     +" stream method signatures. method(" + methodName + ")");
         });
     }
+    
+    public static boolean isProtobufClass(Class<?> clazz) {
+        while (clazz != Object.class && clazz != null) {
+            Class<?>[] interfaces = clazz.getInterfaces();
+            if (interfaces.length > 0) {
+                for (Class<?> clazzInterface : interfaces) {
+                    if (PROTOBUF_MESSAGE_CLASS_NAME.equalsIgnoreCase(clazzInterface.getName())) {
```

Comment:
`equals` is enough?

Reply:
I think the authors of this method wanted to do this without introducing protobuf dependencies,otherwise we can use isAssignableFrom to judge pb class

Reply:
bingo